goalc/compiler/compilation/Asm.cpp

```diff
@@ -272,6 +272,22 @@ Val* Compiler::compile_asm_mov(const goos::Object& form, const goos::Object& res
   return get_none();
 }
 
+Val* Compiler::compile_asm_slt(const goos::Object& form, const goos::Object& rest, Env* env) {
```

Comment:
It looks like you ended up implementing it as a macro instead, maybe we should remove this function